pyfar/classes/audio.py

```diff
@@ -1249,3 +1249,85 @@ def _divide(a, b):
 
 def _power(a, b):
     return a**b
+
+
+def _match_fft_norm(fft_norm_1, fft_norm_2, division=False):
+    """
+    Helper function to determine the fft_norm resulting from an
+    arithmetic operation of two signals.
```

Comment:
Also works for Time/FrequencyData :)
```suggestion
    arithmetic operation of two audio objects.
```

Reply:
totally right, thx. :)